util/election2/etcd/client.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package etcd
+
+import (
+	"time"
+
+	"github.com/coreos/etcd/clientv3"
+)
+
+// NewClient returns an etcd client connecting to the passed in servers'
+// endpoints, with the specified dialing timeout.
```

Comment:
I'm not sure this comment would help someone new understand what's needed here and why.  

Could we point out more clearly that this creates a `clientv3.Client` object from the etcd package in the Trillian `vendor/` directory, which allows external clients/codebases to build an object that matches the Trillian internal implementation (a `clientv3.Client` built from a different codebase/location, even if it's the same code, wouldn't have the required matching type).

Reply:
Done.